Tests/test_image_convert.py

```diff
@@ -93,7 +93,7 @@ def test_trns_p(tmp_path):
     f = str(tmp_path / "temp.png")
 
     im_l = im.convert("L")
-    assert im_l.info["transparency"] == 0  # undone
+    assert im_l.info["transparency"] == 1  # undone
```

Comment:
The first palette entry is `(0, 1, 2)`. `(0*299 + 1*587 + 2*114) / 1000` = 0.815, so this is now rounded to 1.